lib/wallet.js

```diff
@@ -484,6 +484,22 @@ class WalletClient extends Client {
     return this.post(`/wallet/${id}/create`, options);
   }
 
+  /**
+   * Create pre-signed bid and reveal txs,
+   * fill, and optionally sign and broadcast.
+   * @param {Object} options
+   * @param {String} options.name
+   * @param {Number} options.bid
+   * @param {Number} options.lockup
```

Comment:
might need to include `passphrase` as well, I see that as an option in hsd wallet/http.js

Reply:
Added passphrase